src/ckeditor.js

```diff
@@ -10,7 +10,7 @@ import HorizontalLine from "@ckeditor/ckeditor5-horizontal-line/src/horizontalli
 import Italic from "@ckeditor/ckeditor5-basic-styles/src/italic.js";
 import Link from "@ckeditor/ckeditor5-link/src/link.js";
 import List from "@ckeditor/ckeditor5-list/src/list.js";
-import Mathematics from 'ckeditor5-math/src/math';
+import Mathematics from "@isaul32/ckeditor5-math/src/math";
```

Comment:
Is there a specific reason why take that from a specific person instead of directly from ckeditor?

Reply:
Yeap, the math plugin provided by CK itself is the one we decided against, as it is one of the premium plugins.

Reply:
for any reason they shifted this after version 36.0.2 to new npm registry entry